theta_kappa.dot

```diff
@@ -780,12 +909,12 @@ digraph "Theta Kappa Zeta Family Tree" {
   tk921 [label="TK921|Brock Jason Rule"] tk921 -> tk946
   tk920 [label="TK920|Mike Rasmussen"] tk920 -> tk1020, tk968
   tk919 [label="TK919|Timothy Dean Plassmeier"]
-  tk918 [label="TK918|John Charle Pfeifer"] tk918 -> tk983, tk1021
+  tk918 [label="TK918|John Charle Pheifer"] tk918 -> tk983, tk1021
   /*tk917 [label="TK917|Michael Wayne Persinger"]*/
   /*tk916 [label="TK916|Scott Alan Orr"]*/
   tk915 [label="TK915|Richard Joseph Nyikos"]
   tk914 [label="TK914|Mike Miller"] tk914 -> tk976, tk943
-  tk913 [label="TK913|Mike McLear"] tk913 -> tk965
+  tk913 [label="TK913|Mike McLear"] tk913 -> tk965; tk913 -> tk1787 [style=dotted]
```

Comment:
Tell me about this. What does this mean?

Reply:
Because the next little is so far down, rather than just having a straight line going through everything, there's a dotted line showing the connection, need to have the whole tk913 -> tk1787 to specify different style. At least, that's what I think. A lot of these changes were made by Chris, not me.